exonum-java-binding/core/src/main/java/com/exonum/binding/core/storage/database/TemporaryDb.java

```diff
@@ -24,26 +24,30 @@
 import com.google.common.annotations.VisibleForTesting;
 
 /**
- * An in-memory database for testing purposes. It can create both read-only snapshots
+ * A wrapper over the `RocksDB` backend which stores data in the temporary
```

Comment:
The RocksDB seems to be an implementation detail. It is a "A MerkleDB which stores its data in the temporary directory for testing purposes.", isn't it?

Reply:
Should MerkleDB be mentioned there?

Reply:
It is named as such in the main documentation, IIRC, and Exonum uses its abstractions (= the indexes) directly in its public API.